specification.md

```diff
@@ -3672,6 +3691,12 @@ interface Capabilities {
    * `OutputEvent.output` and `Variable.value` fields.
    */
   supportsANSIStyling?: boolean;
+
+  /**
+   * The debug adapter supports the `instructionReference` and `offset` fields
+   * in the `gotoTargets` request.
+   */
+  supportsGotoInstruction?: boolean;
```

Comment:
I would name this `supportsGotoInstructionTargets` just in case we ever have another type of 'go to' action